htdocs/core/modules/modTakePos.class.php

```diff
@@ -266,9 +267,45 @@ public function __construct($db)
 	 */
 	public function init($options = '')
 	{
-		global $conf, $db;
+		global $conf, $db, $langs, $user;
+		$langs->load("cashdesk");
 
 		dolibarr_set_const($db, "TAKEPOS_PRINT_METHOD", "browser", 'chaine', 0, '', $conf->entity);
+		
+		//Default customer for Point of sale
+		if (empty(getDolGlobalInt('CASHDESK_ID_THIRDPARTY1'))) {
+			$societe = new Societe($db);
+			$societe->nom = $langs->trans("DefaultPOSThirdLabel");
```

Comment:
What if you disable and reenable the module. You must check the objectvdoes not existvyet by a fetch to avoid to try to create it a second time

Reply:
When reactivated the second time, as the variable CASHDESK_ID_THIRDPARTY1 is already assigned, it will not be created again. It is only created the first time when the CASHDESK_ID_THIRDPARTY1 variable has never been initialized.